skimage/feature/tests/test_peak.py

```diff
@@ -272,14 +312,14 @@ def test_adjacent_different_objects():
 
 
 def test_four_quadrants():
-    image = np.random.uniform(size=(40, 60))
-    i, j = np.mgrid[0:40, 0:60]
-    labels = 1 + (i >= 20) + (j >= 30) * 2
+    image = np.random.uniform(size=(20, 30))
```

Comment:
I guess that should be fine?

On Fri, 27 May 2016 at 11:02 François Boulogne <notifications@github.com>
wrote:

> In skimage/feature/tests/test_peak.py
> <https://github.com/scikit-image/scikit-image/pull/2098#discussion_r64944523>
> :
>
> > @@ -272,14 +312,14 @@ def test_adjacent_different_objects():
> >
> >
> >  def test_four_quadrants():
> > -    image = np.random.uniform(size=(40, 60))
> > -    i, j = np.mgrid[0:40, 0:60]
> > -    labels = 1 + (i >= 20) + (j >= 30) * 2
> > +    image = np.random.uniform(size=(20, 30))
>
> It's defined globally at line 8, should be enough no?
>
> —
> You are receiving this because you are subscribed to this thread.
>
> Reply to this email directly, view it on GitHub
> <https://github.com/scikit-image/scikit-image/pull/2098/files/b84af59c301beb2ce04639d22b4227b229c94493#r64944523>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AACwD6yqFD9XysFbFgA1XTY1NqgjQWRSks5qFzGlgaJpZM4Ijx7J>
> .
>